lib/compilers/argument-parsers.ts

```diff
@@ -702,7 +703,21 @@ export class JavaParser extends BaseParser {
 }
 
 export class ClojureParser extends BaseParser {
+    // Get help line from wrapper not Clojure runtime
+    override async getOptions(helpArg: string) {
+        const clojureCompiler = this.compiler as ClojureCompiler;
+        const result = await this.compiler.execCompilerCached(this.compiler.compiler.exe, [
+            '-M', clojureCompiler.compilerWrapperPath,
+            helpArg,
+        ]);
+        const optionFinder = /^\s*(--.*) - (.*)$/i;
+        const options = result.code === 0 ? this.parseLines(result.stdout + result.stderr, optionFinder) : {};
+        this.compiler.possibleArguments.populateOptions(options);
+        return options;
+    }
+
     override async parse() {
+        await this.getOptions('--help');
```

Comment:
Where is the set of options generated here in the `ClojureParser` supposed to be visible in the UI?

Reply:
These are used during startup and are used by some compilers to tailor what they do if certain command-line flags are specified (e.g. gcc uses it to see if things like "--ansi-color" are ok to use). It _probably_ doesn't matter for Clojure.

Reply:
Ah, ok, reverted in [ded8737](https://github.com/compiler-explorer/compiler-explorer/pull/8146/commits/ded873753e71b08f3af44c824869d27164f899df).